src/main/kotlin/com/atlassian/performance/tools/report/api/result/StatsMeter.kt

```diff
@@ -10,6 +11,45 @@ import java.time.Duration
 
 class StatsMeter {
 
+    private companion object {
+        private val DEFAULT_OUTLIER: OutlierTrimming = OutlierTrimming(0.0, 1.0)
```

Comment:
👌

Reply:
Technically it is a default (outlier) trimming not default outlier.